examples/predator_prey_fast.jl

```diff
@@ -150,12 +148,10 @@ end
 # additional energy and the grass will not be available for consumption until regrowth time
 # has elapsed. If a wolf eats a sheep, the sheep dies and the wolf acquires more energy.
 
-function sheep_eat!(sheep, grass_array, model)
-    isempty(grass_array) && return
-    grass = grass_array[1]
-    if grass.fully_grown
+function sheep_eat!(sheep, model)
+    if model.fully_grown[sheep.pos...]
         sheep.energy += sheep.Δenergy
-        grass.fully_grown = false
+        model.fully_grown[sheep.pos...] = false
```

Comment:
I think splatting `...` is bad for performance and should be replaced with `pos[1], pos[2]`. Not sure though...

Reply:
Nah, there's no difference. It's fine to change if you want to put it in line with the rest though.

Reply:
okay if there is no difference then I'd rather re-write other parts where I write explicitly `pos[1], pos[2]`.